apps/nextjs-website/src/components/Editorial.tsx

```diff
@@ -16,7 +17,15 @@ const makeEditorialProps = ({
 }: EditorialSection): EditorialProps => ({
   ...(eyelet && { eyelet }),
   body: MarkdownRenderer({ markdown: body, variant: 'body2' }),
-  image: <img src={image.url} alt={image.alternativeText ?? undefined} />,
+  image: (
+    <Image
+      src={image.url}
+      alt={image.alternativeText ?? ''}
+      // Needed by Image, will be overwritten
+      width={0}
+      height={0}
```

Comment:
are these properties needed also if the properties are optional? 🤔

Reply:
There needs to be either the both of them (width, height) or the single `fill` property set to true.

Unfortunately we can't set the fill property because it clashes with styles coming from our theme.

Next throws an error saying that Image cannot have `fill={true}` because a `style.height` property is set, so this is the only other option.

The 0 values are so that we aren't giving random arbitrary values and it's going to be obvious if an Image hasn't had its width or height set because it will go down to zero, allowing us to see it immediately when developing/testing.

Reply:
Without that part, you'll see this:

`Error: Image with src "/uploads/151089_7e6fa8785f.jpg" is missing required "width" property.`

In this case you'll see "uploads/151089_7e6fa8785f.jpg" 'cause is the name of the image I uploaded locally